bitnami/mysql/README.md

```diff
@@ -44,35 +43,80 @@ The command removes all the Kubernetes components associated with the chart and
+| `slave.readinessProbe.enabled`            | Turn on and off readiness probe (slave)             | `true`                                                            |
+| `slave.readinessProbe.initialDelaySeconds`| Delay before readiness probe is initiated (slave)   | `15`                                                              |
+| `slave.readinessProbe.periodSeconds`      | How often to perform the probe (slave)              | `10`                                                              |
+| `slave.readinessProbe.timeoutSeconds`     | When the probe times out (slave)                    | `1`                                                               |
+| `slave.readinessProbe.successThreshold`   | Minimum consecutive successes for the probe (slave) | `1`                                                               |
+| `slave.readinessProbe.failureThreshold`   | Minimum consecutive failures for the probe (slave)  | `3`                                                               |
+| `metrics.enabled`                         | Start a side-car prometheus exporter                | `false`                                                           |
+| `metrics.image`                           | Exporter image name                                 | `prom/mysqld-exporter`                                            |
+| `metrics.imageTag`                        | Exporter image tag                                  | `v0.10.0`                                                         |
+| `metrics.imagePullPolicy`                 | Exporter image pull policy                          | `IfNotPresent`                                                    |
+| `metrics.resources`                       | Exporter resource requests/limit                    | `nil`                                                             |
 
 The above parameters map to the env variables defined in [bitnami/mysql](http://github.com/bitnami/bitnami-docker-mysql). For more information please refer to the [bitnami/mysql](http://github.com/bitnami/bitnami-docker-mysql) image documentation.
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```bash
 $ helm install --name my-release \
-  --set mysqlRootPassword=secretpassword,mysqlUser=my-user,mysqlPassword=my-password,mysqlDatabase=my-database \
+  --set root.password=secretpassword,user.database=app_database \
     bitnami/mysql
 ```
 
-The above command sets the MySQL `root` account password to `secretpassword`. Additionally it creates a standard database user named `my-user`, with the password `my-password`, who has access to a database named `my-database`.
+The above command sets the MySQL `root` account password to `secretpassword`. Additionally it creates a database named `my_database`.
```

Comment:
There is discrepancy between the command and the explanation related to the database